atd-vze/src/views/Crashes/VictimNameField.js

```diff
@@ -18,30 +15,17 @@ const VictimNameField = ({
   editRow,
   row,
   field,
+  refetch,
   ...props
 }) => {
-  const crashId = props.match.params.id;
-
-  const personId =
-    tableName === "atd_txdot_primaryperson"
-      ? row.primaryperson_id
-      : row.person_id;
-
-  const { data, refetch } = useQuery(GET_PERSON_NAMES, {
```

Comment:
dont need to query for just the person names, we get them from the main `people_list_view` now